config/zones/CO.yaml

```diff
@@ -1,2 +1,15 @@
 flag_file_name: co.png
 timezone: America/Bogota
+capacity:
+  biomass: 397
+  coal: 1612
+  gas: 2491
+  hydro: 12549
+  nuclear: 0
+  oil: 1346
+  solar: 297
+  wind: 18
+contributors:
+  - alixunderplatz
+parsers:
+  consumption: CO.fetch_consumption
```

Comment:
would you be so kind to run this for me? :-) after all thes years working on the map, I still can't understand how and why^^ :-P

Reply:
Sure thing! The why is simple, when we split up the massive json file into all the yaml files so it would be easier to edit and such we also created a script that generated a json file with the keys used in the frontend. But currently it must be generated manually but #5082 fixes that.

But I'll update the files accordingly.